package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sia",
-  "version": "1.16.0",
+  "version": "1.20.0",
```

Comment:
We're currently at version `1.19.0`. Why do we need to update this number? We're not publishing this application as a private or public NPM package, so the version number has no meaning.

Reply:
We don't need the version number in `package.json` to be up to date; we're not publishing the package nor will we ever.